app.go

```diff
@@ -107,14 +107,19 @@ func (a *App) CreateMagicLink(createMagicLinkBody CreateMagicLinkBody) (*MagicLi
 	}
 
 	var errorText string
+	var errorCode string
```

Comment:
could this also be set for `GetApp` as well so that its error struct is populated for those that are not ready to move to the major version yet?